db/models/comments.model.js

```diff
@@ -33,4 +35,18 @@ exports.addComment = (article_id, username, body) => {
     return db.query(sqlQuery, [article_id, username, body]).then(({ rows }) => {
         return rows;
     })
-}
\ No newline at end of file
+}
+
+exports.deleteComment = async (comment_id) => {
+
+    await checkExists('comments', 'comment_id', comment_id);
+
+    const sqlQuery = `
+        DELETE FROM comments 
+        WHERE comment_id = $1
+        RETURNING *;`;
+
+   await db.query(sqlQuery, [comment_id]).then(({ rows }) => {
+        return rows;
+    })
+}
```

Comment:
Nice use of async/await! 

My advice would be - and it's fine at this stage if you are still trying out both kinds of syntax (async/await and promises) - but at some point I would pick one form and use it throughout your app - including controllers and models and error handlers. It will look more like a decision you made rather than learning!